backend/inspirehep/disambiguation/tasks.py

```diff
@@ -64,3 +68,36 @@ def disambiguate_signatures(self, clusters):
             disambiguation_assigned_clusters.labels("2+").inc()
             LOGGER.debug("Received cluster with more than 1 author.")
     db.session.commit()
+
+
+@shared_task(ignore_result=False, bind=True)
+def disambiguate_authors(self, record_uuid):
```

Comment:
if there are changes to the linked authors in this task, then the record will be modified, so the task will fire again (and hopefully there will be no changes the second time around). Maybe we want to prevent this second task from being triggered somehow (this might be also be done in a future story, as it's not part of this one)? @drjova

Reply:
good point! Yes, let's have this merged so we can plan for the next steps